src/app/authentication/authentication.component.html

```diff
@@ -1 +1,27 @@
-<ng-content></ng-content>
+<div class="login-container">
+  <form [formGroup]="authForm" (ngSubmit)="onSubmit(authForm.value)" [ngClass]="{'login-form': true, 'login-form-sign-up': !isLogin}" >
```

Comment:
when use reactive form don't need pass value on function 
```suggestion
  <form [formGroup]="authForm" (ngSubmit)="onSubmit()" [ngClass]="{'login-form': true, 'login-form-sign-up': !isLogin}" >
```